iwyu_path_util.cc

```diff
@@ -178,9 +177,13 @@ string GetParentPath(const string& path) {
 
 // Converts a file-path, such as /usr/include/stdio.h, to a
 // quoted include, such as <stdio.h>.
-string ConvertToQuotedInclude(const string& filepath) {
+string ConvertToQuotedInclude(const string& filepath, const string& includerPath) {
+  if (filepath == "<built-in>")
+    return filepath;
+
   // First, get rid of leading ./'s and the like.
   string path = NormalizeFilePath(filepath);
+  const bool isAbsolute = IsAbsolutePath(path);
```

Comment:
You probably want to name this so it's clearer what it refers to, e.g. `isIncludeeAbsolute`